README.md

```diff
@@ -1,17 +1,17 @@
 # AI Ecosystem Extension Specification
 
+This document explains the AI Ecosystem Extension to the
```

Comment:
If we want this generally at the beginning, we should update the template and all extensions accordingly. But I think it would be easier to keep it as it is. It works good in STAC.